Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -437,6 +436,23 @@ public class CarPlayNavigationViewController: UIViewController {
         let alert = CPActionSheetTemplate(title: arrivalTitle, message: arrivalMessage, actions: [rateAction, exitAction])
         carInterfaceController.presentTemplate(alert, animated: true)
     }
+    
+    func presentWaypointArrivalUI(for waypoint: Waypoint) {
```

Comment:
Was this method already available in previous versions of Navigation SDK, but was removed at some point?

Reply:
Yes, it was removed at the same time when the arrival UI was removed.